### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by sougou**
* Revert logo color to the not so blue one
* Change theme color to match logo color
* Add "who uses vitess" user logos
* Add YouTube presentation video

### CODE REVIEW

**Comment by bbeaudreault** [comment]
👍

File: vitess.io/images/vitess_logo.svg [hunk 1]
```diff
@@ -152,125 +152,125 @@
          d="m 367.1282,648.85881 -27.87054,-51.05696 43.91222,-28.31214 z"
```
Comment (michael-berlin) [inline vitess.io/images/vitess_logo.svg]: Can you please re-create the favicon as well?

It looks like I used Inkscape to open the .svg file and then export it to .png.

https://github.com/youtube/vitess/commit/3d4bf0d3c268728144cdf86db159fc62616203f1#diff-2901c8251c80036790228c936bc5a45a

File: vitess.io/css/site.scss [hunk 2]
```diff
@@ -5,12 +5,12 @@
 
 // Colors
 // We're using the Material "Indigo" palette: https://material.google.com/style/color.html#color-color-palette
```
Comment (michael-berlin) [inline vitess.io/css/site.scss]: I think you can remove this comment.

Can you please instead comment why you chose these colors and where you got them from?

**Comment by sougou** [comment]
Preview site updated. Please see comments in the commit on how I've addressed them.

**Comment by michael-berlin** [comment]
LGTM

Updates look great. As discussed offline, can you please change the background color from Cyan 100 to Cyan 50?

btw: You should be able to run `./vitess.io/publish-site.sh` on your machine within this branch to re-generate the pages. Then you can push the generated files as part of this branch and this way merge them together with the Markdown changes.

**Pull request merged**